recipes/sdl2_ttf/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "SDL2_ttf" %}
+{% set version = "2.0.14" %}
+{% set sha256 = "34db5e20bcf64e7071fe9ae25acaa7d72bdc4f11ab3ce59acc768ab62fe39276" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn:  {{ name }}-{{ version }}.tar.gz
+  url:  https://www.libsdl.org/projects/SDL_ttf/release/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  features:
+    - vc9  # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py35]
+
+requirements:
+  build:
+    - python  # [win]
+    - cmake  # [win]
+    - vc 9  # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py35]
+    - autoconf  # [unix]
+    - automake  # [unix]
+    - libtool  # [unix]
+    - toolchain
+    - sdl2
```

Comment:
It should be something like `sdl2 2.0.*`, but since updates to this library are very rare or infrequent, I think not pinning it will pose any problems. I haven't pinned it in the other recipes depending on sdl2 either in any case.